app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,2 @@
+class PostsController < ApplicationController
+end
```

Comment:
- [ ] Please make sure that posts controller has #index and #show actions defined.